templates/telco/index.html

```diff
@@ -0,0 +1,242 @@
+                            url="https://assets.ubuntu.com/v1/553512df-Anbox_Telco_Video.svg",
+                            alt="",
+                            width="190",
+                            height="190",
+                            hi_def=True,
+                            loading="auto"
+                            ) | safe
+                        }}
+                    </div>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-6">
+                    <h2>Virtual mobile infrastructure <br>for RANs</h2>
+                </div>
+            </div>
+            <div class="row u-equal-height">
+                <div class="col-6 p-card">
+                    <h3>Scalability and high-density</h3>
+                    <hr>
+                    <p>Anbox Cloud scales vertically with a high container density on servers. The platform also scales horizontally across edge data centers. Scalability allows telco service providers to instantly match compute capacity to subscriber demands. High container density keeps the unit economics low.</p>
+                </div>
+                <div class="col-6 p-card">
+                    <h3>Infrastructure automation</h3>
+                    <hr>
+                    <p>The key components that make up Anbox Cloud are deployed and orchestrated automatically on server infrastructure. Automation reduces the cost of operations for telco service providers using Anbox Cloud. Anbox Cloud is an easily deployed platform that will be operated at minimal effort by telco teams.</p>
+                </div>
+            </div>
+            <div class="row u-equal-height">
+                <div class="col-6 p-card">
+                    <h3>Simple client-side integration</h3>
+                    <hr>
+                    <p>Anbox Cloud is easy to integrate into mobile clients. Graphical output is streamed from the cloud to mobile devices. Sensor inputs and output from mobile devices are synchronised with Android virtual machines in the cloud. This data exchange is based on the WebRTC protocol, which makes mobile client development simple.</p>
+                </div>
+                <div class="col-6 p-card">
+                    <h3>Cloud-side latency optimisation</h3>
+                    <hr>
+                    <p>Anbox Cloud leverages hardware acceleration for rich graphical rendering, animations and effects. The platform also enables high-performance video encoding and decoding and streaming. To minimise latency bottlenecks, compute intensive tasks like encoding and decoding are pooled on GPUs on the server side to offload mobile devices.</p>
+                </div>
+            </div>
+        </section>
+
+        <section>
+            <div class="p-strip--light u-no-padding--bottom">
+                <div class="row">
+                    <div class="col-6">
+                        <h2>What’s under the hood</h2>
```

Comment:
(we use straight quotes/apostrophes)
```suggestion
                        <h2>What's under the hood</h2>
```